Resources/Prototypes/Species/arachnid.yml

```
@@ -59,11 +59,11 @@
       points: 1
       required: false
     HeadSide:
-      points: 3
+      points: 6
       required: true
       defaultMarkings: [ ArachnidCheliceraeDownwards ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)